src/tun.cc

```diff
@@ -127,34 +126,43 @@ uint16_t Tun::Read(const IOType &type, char *buf, uint16_t len) {
   assert(nread > 0);
   return nread;
 }
-
-uint16_t Tun::Read(const vector<IOType> &type_arr, char *buf, uint16_t len, IOType *type_out) {
+/*
+uint16_t Tun::Read(const vector<IOType> &type_arr, char *buf, uint16_t len, IOType *type_out, int *radio_id) {
```

Comment:
Please remove all the unused functions. We can find them from previous commits if needed.